tests/migrations/test_commands.py

```diff
@@ -381,8 +381,10 @@ def test_showmigrations_no_migrations(self):
 
     @override_settings(INSTALLED_APPS=['migrations.migrations_test_apps.unmigrated_app'])
     def test_showmigrations_unmigrated_app(self):
-        with self.assertRaisesMessage(CommandError, 'No migrations present for: unmigrated_app'):
-            call_command('showmigrations', 'unmigrated_app')
+        err = io.StringIO()
+        with self.assertRaises(SystemExit):
+            call_command('showmigrations', 'unmigrated_app', stderr=err)
+        self.assertIn('No migrations present for: unmigrated_app', err.getvalue())
```

Comment:
This should be `self.assertEqual('migrations\n (no migrations)\n', out.getvalue().lower())` as in the above test (no SystemExit).